2017/Team-Training/Devspace/index.html

```diff
@@ -0,0 +1,673 @@
+                </section>
+                <section>
+                    <h2>First round</h2>
+                    <ul>
+                        <li>Port dynamically assigned e.g. Jenkins UI no longer available at <a href="">https://devspace_openstack_ip:8443</a> <a href="https://github.com/openmicroscopy/devspace/pull/74" target="_blank"> PR #74 (merged)</a></li>
+                        <li>Jenkins dependencies: upgrade<a href="https://github.com/openmicroscopy/devspace/pull/77" target="_blank">PR #77</a></li>
+                        <li>No longer share keys and token</li>
+                        <li>Use <a href="https://github.com/openmicroscopy/devspace" target="_blank">devspace repository </a> for instructions and playbooks</li>
+                        <li>Simplify set up when possible</li>
+                    </ul>
+                </section>
+                 <section>
+                    <h1>Work in progress: version 0.6.0</h1>
+                </section>
+                <section>
+                    <h2>Reviewer wanted!</h2>
+                    Deployment on Openstack is based on
+                    <ul>
+                        <li><a href="https://github.com/openmicroscopy/devspace/pull/81" target="_blank">Devspace PR #81 </a>
+                        </li>
+                        <li><a href="https://github.com/openmicroscopy/ansible-role-devspace/pull/5" target="_blank">ansible-role-devspace PR #5 </a>
+                        </li>
+                    </ul>
+                </section>
+                <section>
+                    <h2>Configuration: Openstack</h2>
+                    <ul>
+                        <li>Connect <a href="https://pony.openmicroscopy.org" target="_blank">OME OpenStack</a></li>
+                        <li>Go to <code>Access & Security > Key Pairs</code>, register a key</li> 
+                        <li>Go to <code>Access & Security > API Access</code>, download your configuration by clicking on <code>Download OpenStack RC File v2.0</code>. The file will be named by default <code>omedev-openrc.sh</code></li>
+                    </ul>
+                </section>
+                <section>
+                    <h2>Configuration: SSH and Git <i>(NEW)</i></h2>
+                    <ul>
+                        <li>Create a new <code>id_gh_rsa</code> SSH key <b>without</b> passphrase
+                            in your <code>.ssh</code> directory
+                            <pre><code style="width:900px">
+ssh-keygen -t rsa -b 4096 -C "your_email_address" -f ~/.ssh/id_gh_rsa -q -P ""
+                            </code></pre>                            
+                        </li>
+                        <li>Upload the public key i.e. <code>id_gh_rsa.pub</code> to your GitHub account</li>
+                        <li>Open  <code>.ssh/config</code> and add the following:
+<pre><code style="width:900px">
+Host github.com
+    User git
+    IdentityFile ~/.ssh/id_gh_rsa
```

Comment:
@pwalczysko experienced some issue as his `.ssh/config` contained key/values that were specific to OSX.

Reply:
I discussed that with @pwalczysko today and we agreed that we will need an extra folder
as you mentioned above to avoid conflicts